engineering-team.html

```diff
@@ -157,9 +168,38 @@ <h3>Elixir</h3>
 <!-- Script for static navbar and footer -->
 <script src="assets/js/navigation.js"></script>
 <script src="assets/js/functions.js"></script>
+<!--Mustache.js-->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/mustache.js/2.3.0/mustache.min.js"></script>
+<script type="text/html" id="template-posts">
+    <div class="row ">
+        {{#data}}
+        <div class="col-md-4 mb-5" >
+            <div class="card px-0 ">
+                <a href="{{url}}" target="_blank" class="card shadow project-card">
+                    <div class="card-body posts-card-body">
+                        <img src="{{image}}" class="img-fluid  img-full" alt="Rounded image">
+                        <p class="card-text text-darker text-left ml-3">
+                            {{title}}
+                        </p>
+                    </div>
+                    <div class="card-footer">
+                        <div class="author">
+                            <img src="{{author_image}}" alt="..." class="avatar img-raised">
+                            <span>{{author_name}}</span>
+                        </div>
+                    </div>
+                </a>
+            </div>
+        </div>
+        {{/data}}
+    </div>
+</script>
 <script>
     $(function () {
-        loadNavAndFooter('assets/content/static')
+        loadNavAndFooter('assets/content/static');
+        // loadBlogPosts();
+        let url = "https://script.google.com/macros/s/AKfycbyUdfhk9WWFLoMWxQvNIEUNl793RbXGnyo-ZIYRn9K1T6QFN1_9G8Ln9NpTCCJba2dzOg/exec";
+        loadPosts(url);
```

Comment:
```suggestion
        loadPosts('https://script.google.com/macros/s/AKfycbyUdfhk9WWFLoMWxQvNIEUNl793RbXGnyo-ZIYRn9K1T6QFN1_9G8Ln9NpTCCJba2dzOg/exec');
```

Reply:
This change should be done to the all changed pages right?